packages/botonic-core/src/core-bot.js

```diff
@@ -44,6 +44,10 @@ export class CoreBot {
     session.__locale = locale
   }
 
+  getPlugins() {
```

Comment:
Why is this method necessary? `plugins` is a public value of the CoreBot.

Reply:
yes, i thought that maybe would be clearer with a getter, but as it's public, makes sense we can retrieve it directly, changed 👍